spec/requests/api/donor_can_create_foodbag_spec.rb

```diff
@@ -0,0 +1,89 @@
+RSpec.describe 'POST /api/foodbags', type: :request do
+  let(:donor) { create(:user) }
+  let(:headers) { donor.create_new_auth_token }
+  describe 'successfully create a foodbag' do
+    before do
+      post '/api/foodbags',
+           params: {
+             foodbag: {
+               pickuptime: 'morning',
+               status: 'available',
+               donor_id: donor.id
```

Comment:
you don't need to send in the donor_id as a param.... because it is being saved on that user directly

Reply:
fixed. we thought it was need as a params, have done so in previous api repos...